### DISCUSSION TITLE
Lineage 19.1 sync issue

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#8779] [General]
i am getting sync error while trying to build lineage 19.1, tried 4-5 times since yesturday. 

Build Link: https://cirrus-ci.com/task/5704464878272512
                  https://cirrus-ci.com/task/5793086159716352

**Comment by ItsAdityan** [comment]
Provide build link

**Comment by RahulPalXDA** [comment]
> i am getting sync error while trying to build lineage 19.1, tried 4-5 times since yesturday.
> 
> Build Link: https://cirrus-ci.com/task/5704464878272512 https://cirrus-ci.com/task/5793086159716352

Sir it's clearly showing somthing probelm in your local manifest. Fix that, that can cure that error.
